.npmrc

```diff
@@ -0,0 +1 @@
+auto-install-peers=true
```

Comment:
@rjchow Could you add a comment to explain why we need this? Thank you!

>  Lines in .npmrc files are interpreted as comments when they begin with a ; or # character. .npmrc files are parsed by [npm/ini](https://github.com/npm/ini), which specifies this comment syntax.

Reply:
*done!